buildSrc/src/main/groovy/org/elasticsearch/gradle/vagrant/VagrantTestPlugin.groovy

```diff
@@ -154,20 +156,39 @@ class VagrantTestPlugin implements Plugin<Project> {
     }
 
     private static void createPrepareVagrantTestEnvTask(Project project) {
-        File batsDir = new File("${project.buildDir}/${BATS}")
+        File packagingDir = new File(project.buildDir, PACKAGING_CONFIGURATION)
+        Task createPackagingDirTask = project.tasks.create('creatingPackagingDir')
```

Comment:
Why do we need this task? Copy tasks will create the destination dir if it doesn't exist. Worst case, this could be a doFirst on copyPackagingArchives.

Reply:
You're right, we don't. I inferred from the `createBatsDirs` task that it had to already exist - I'll remove both